app/main.py

```diff
@@ -2,0 +1,18 @@
+import os
+import shutil
+
+
+def move_file(command: str) -> None:
+    if (command[:3]) != "mv " or command.count(" ") != 2:
+        return
+    _, old_file, new_file = command.split(" ")
+
+    if new_file.count("/") != 0:
+        *path, new_file_name = new_file.split("/")
+        print(path)
```

Comment:
The `print(path)` statement seems to be used for debugging purposes. If it's not needed for the final implementation, consider removing it to clean up the output.